__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const seed = require('../db/seeds/seed.js');
+const {articleData, commentData, topicData, userData}   = require('../db/data/test-data');
+const db = require('../db/connection');
+const request = require('supertest');
+const app = require('../db/app.js');
+
+beforeEach(() => {
+    return seed({articleData, commentData, topicData, userData});
+  });
+  
+  afterAll(() => db.end());
+
+  describe('NC-News',() => {
+    describe('Get api/topics', () => {
+      test('Responds with 200 and array of objects', () => {
+return request(app).get("/api/topics").expect(200).then(({body: {topics}}) => {
```

Comment:
I would love to see another assertion here checking that the `topics` array that comes back is the right length too so we know it's got everything we want